.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,59 @@
+---
+name: 🐛 Bug report
+about: Create a report to help us improve
+title: '🐛 '
+labels: 'bug'
+assignees: ''
+
+---
+<!---
+Please DO NOT remove any fields from this template. If there is nothing to add, leave N/A.
+--->
+### Operator Version, Kind and Kubernetes Version
+ - Operator version:
+ - Kind: Workspace | Module
+ - Kubernetes version:
+
+### YAML Manifest File
+```yaml
+# Copy-paste your YAML manifest here
+```
+
+### Output Log
+<!---
+Please provider a link to a GitHub Gist containing the complete output log.
+Please DO NOT paste the debug output in the issue; just paste a link to the Gist.
+--->
+
+### Kubectl Outputs
+<!---
+Please provider a link to a GitHub Gist containing the complete kubectl outputs.
+Please DO NOT paste the kubectl output in the issue; just paste a link to the Gist.
+--->
+
+### Steps To Reproduce
+Steps to reproduce the behavior:
```

Comment:
Is this an example of the actual recommended steps required of the user to perform?
Let's mark as "example" if that is the case.